moped-editor/src/components/GridTable/Filters.js

```diff
@@ -163,7 +223,19 @@ const Filters = ({
    * @function setFilterParameters - Update the state of filterParameters
    * @default {filters}
    */
-  const [filterParameters, setFilterParameters] = useState(filters);
+  const initialFilterParameters = useMemo(() => {
```

Comment:
Since we aren't storing the full filter configuration in the url anymore, we need to build it for any existing filters in the url.